gazebodistro/collection-garden.yaml

```diff
@@ -15,15 +15,15 @@ repositories:
   ign-gazebo:
     type: git
     url: https://github.com/ignitionrobotics/ign-gazebo
-    version: ign-gazebo6
+    version: ign-gazebo7
```

Comment:
These could be `main`, but I think this file isn't used for anything, so this should be ok

Reply:
I'll try to see where this happens in `bump_dependency.bash`...

Reply:
It's a blind search-and-replace. I don't think it's worth "fixing"... We often forget to flip from `main` to release branches after the official release, so maybe it's good to be this way from the start

Reply:
the homebrew formula test checks that all the branches exist: https://github.com/osrf/homebrew-simulation/blob/master/Formula/ignition-garden.rb#L55-L58

so that may fail until we create the release branches

Reply:
Ahh gotcha, yeah then we need to fix this

Reply:
I'm going to close this and test an improved script to see if that can get it right